clang/include/clang/Driver/Options.td

```diff
@@ -7110,6 +7110,11 @@ defm sycl_decompose_functor
           NegFlag<SetFalse, [], [ClangOption, CLOption], "Do not">,
           BothFlags<[], [ClangOption, CLOption, CC1Option],
            " decompose SYCL functor if possible (experimental, CUDA only)">>;
+defm sycl_cuda_compat
+    : BoolFOption<"sycl-cuda-compatibility", LangOpts<"SYCLCUDACompat">, DefaultFalse,
+          PosFlag<SetTrue, [], [ClangOption, CLOption, DXCOption, CC1Option], "Enable CUDA compatibility mode. "
+          "This implies the definition of CUDA macros and the inclusion of implicit header files.">,
+          NegFlag<SetFalse, [], [ClangOption, CLOption, DXCOption, CC1Option], "Disable CUDA compatibility mode.">>;
```

Comment:
The additional `DXCOption` driver enabling is not necessary - we don't support that driver.